nevergrad/optimization/test_optimizerlib.py

```diff
@@ -462,3 +462,15 @@ def test_bo_ordering() -> None:
     )
     cand = optim.ask()
     optim.tell(cand, 12)
+
+
+@pytest.mark.parametrize(  # type: ignore
+    "name,expected", [("NGOpt2", ["CMA", "CMandAS2"]),
```

Comment:
@teytaud I only made sure it worked with NGOpt2, for now it will not chose the correct optimizer with other versions, because it will not identify choices. Which ones should be updated as well?